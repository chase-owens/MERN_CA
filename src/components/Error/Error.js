import React from 'react';
import { theme } from '../../styles/theme';
import Typography from '@material-ui/core/Typography';

const Error = () => {
  return (
    <div
      style={{
        width: '100vw',
        height: '100vh',
        background: theme.palette.primary.main
      }}
    >
      <Typography variant='display4'>
        Sorry, An error occurred. Error: The dogs ate the code.
      </Typography>
    </div>
  );
};

export default Error;
