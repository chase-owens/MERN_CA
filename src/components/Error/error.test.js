import React from 'react';
import Error from './Error';
import { shallow } from 'enzyme';

describe('Error', () => {
  it('renders without crashing', () => {
    const error = shallow(<Error />);
    expect(error).toMatchSnapshot();
  });
});
