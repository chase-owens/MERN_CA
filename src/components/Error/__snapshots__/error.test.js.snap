// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders without crashing 1`] = `
<div
  style={
    Object {
      "background": "#29bbf6",
      "height": "100vh",
      "width": "100vw",
    }
  }
>
  <WithStyles(Typography)
    variant="display4"
  >
    Sorry, An error occurred. Error: The dogs ate the code.
  </WithStyles(Typography)>
</div>
`;
